TestDeltaLakeRegisterTableProcedureWithFileMetastore fails on master

```
Expecting message:
  "Metadata not found in transaction log for test_delta_lake_register_table_2fje0836o8.test_register_table_with_no_transaction_log_new_axe3j48dug"
to match regex:
  ".*Failed to access table location: (.*)"
but did not.

Throwable that failed the check:

io.trino.testing.QueryFailedException: Metadata not found in transaction log for test_delta_lake_register_table_2fje0836o8.test_register_table_with_no_transaction_log_new_axe3j48dug
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:122)
	at io.trino.testing.DistributedQueryRunner.executeWithQueryId(DistributedQueryRunner.java:493)
	at io.trino.testing.QueryAssertions.assertQueryFails(QueryAssertions.java:439)
	at io.trino.testing.AbstractTestQueryFramework.assertQueryFails(AbstractTestQueryFramework.java:435)
	at io.trino.plugin.deltalake.BaseDeltaLakeRegisterTableProcedureTest.testRegisterTableWithInvalidDeltaTable(BaseDeltaLakeRegisterTableProcedureTest.java:233)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
	Suppressed: java.lang.Exception: SQL: CALL system.register_table('test_delta_lake_register_table_2fje0836o8', 'test_register_table_with_no_transaction_log_new_axe3j48dug', 'file:/tmp/TrinoTest14530672212085892479/delta_lake_data/test_delta_lake_register_table_2fje0836o8/test_register_table_with_no_transaction_log_n2fi7z69m5-e9484ef2f817453b9cd68347f0a8798f')
		at io.trino.testing.DistributedQueryRunner.executeWithQueryId(DistributedQueryRunner.java:497)
		... 16 more
	Suppressed: java.lang.Exception: Query: CALL system.register_table('test_delta_lake_register_table_2fje0836o8', 'test_register_table_with_no_transaction_log_new_axe3j48dug', 'file:/tmp/TrinoTest14530672212085892479/delta_lake_data/test_delta_lake_register_table_2fje0836o8/test_register_table_with_no_transaction_log_n2fi7z69m5-e9484ef2f817453b9cd68347f0a8798f')
		at io.trino.testing.QueryAssertions.assertQueryFails(QueryAssertions.java:448)
		... 15 more
Caused by: io.trino.spi.TrinoException: Metadata not found in transaction log for test_delta_lake_register_table_2fje0836o8.test_register_table_with_no_transaction_log_new_axe3j48dug
	at io.trino.plugin.deltalake.transactionlog.TransactionLogAccess.lambda$getMetadataEntry$4(TransactionLogAccess.java:205)
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at io.trino.plugin.deltalake.transactionlog.TransactionLogAccess.getMetadataEntry(TransactionLogAccess.java:205)
	at io.trino.plugin.deltalake.procedure.RegisterTableProcedure.doRegisterTable(RegisterTableProcedure.java:163)
	at io.trino.plugin.deltalake.procedure.RegisterTableProcedure.registerTable(RegisterTableProcedure.java:116)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:732)
	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:757)
	at io.trino.execution.CallTask.execute(CallTask.java:211)
	at io.trino.execution.CallTask.execute(CallTask.java:71)
	at io.trino.execution.DataDefinitionExecution.start(DataDefinitionExecution.java:146)
	at io.trino.execution.SqlQueryManager.createQuery(SqlQueryManager.java:257)
	at io.trino.dispatcher.LocalDispatchQuery.lambda$startExecution$7(LocalDispatchQuery.java:143)
	at io.trino.$gen.Trino_testversion____20230429_011955_555.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

--- a/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeRegisterTableProcedureTest.java
+++ b/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeRegisterTableProcedureTest.java
@@ -231,7 +231,7 @@ public abstract class BaseDeltaLakeRegisterTableProcedureTest
         new File(getTransactionLogJsonEntryPath(transactionLogDir, 0)).createNewFile();
 
         assertQueryFails(format("CALL system.register_table('%s', '%s', '%s')", SCHEMA, tableNameNew, tableLocation),
-                ".*Failed to access table location: (.*)");
+                ".*Metadata not found in transaction log for (.*)");
 
         deleteRecursively(Path.of(new URI(tableLocation).getPath()), ALLOW_INSECURE);
         metastore.dropTable(SCHEMA, tableName, false);

